pybamm/spatial_methods/finite_volume.py

```diff
@@ -225,6 +263,8 @@ def definite_integral_matrix(self, domain):
         ----------
         domain : list
             The domain(s) of integration
+        vector_type : str, optional
+            Whether to return a row or column vector (defualt is row)
```

Comment:
this isn't an input to `definite_integral_matrix`. I'm not entirely clear on the difference between `definite_integral_vector` and `definite_integral_matrix`?

Reply:
yeah I think I can get rid of `definite_integral_vector` here and just pass the row/column input to `definite_integral_matrix`